dev-db/etcd-wrapped/etcd-wrapped-0.4.9.ebuild

```diff
@@ -0,0 +1 @@
+files/etcd-wrapped.ebuild
```

Comment:
For the sake of not having to maintain an old ebuild I would support etcd:0 with this one.